index.html

```diff
@@ -3137,10 +3136,9 @@ <h3>Exploration Mechanisms</h3>
         <a>TDDs</a> also provide change notification mechanisms 
         to support secure distribution of
         <a>TD</a> updates to authorized consumers.
-        <span class="rfc2119-assertion" id="arch-discovery-metadata-after-authorization">
-        Any <a>WoT Discovery</a> "exploration"
-        implementation MUST only provide metadata and <a>TDs</a>
-        after appropriate best-practice authentication and authorization.</span>
+        To preserve security and privacy, any <a>WoT Discovery</a> "exploration"
```

Comment:
"To preserve security and privacy" it an editorial change; it was added to make the text flow better.